### ISSUE TITLE
Updating the PyTorch Lightning example to >= 2.0

### ISSUE THREAD

**Issue by JustinGoheen** [optuna/optuna-examples#204]
👋 Is it okay if I submit a PR to update the PyTorch Lightning example to use version 2.0?

**Comment by toshihikoyanase** [comment]
Thank you for your interest. Since Optuna v3.3.0 supports PyTorch Lightning 2.0, your contribution to update the example would be very helpful for users.

**Comment by c-bata** [comment]
Just a question, but it seems that [the example](https://github.com/optuna/optuna-examples/blob/main/pytorch/pytorch_lightning_simple.py) is already using the latest version of PyTorch Lightning. Is it that some of the syntax is outdated, or are there any deprecated features being used?

https://github.com/optuna/optuna-examples/blob/e61dedae835adddd4bca45eb219683019bac279a/pytorch/requirements.txt#L4

**Comment by JustinGoheen** [comment]
Yes to the syntax. Essentially I would only need to update to `>=2.0`, run any needed tests to ensure everything is fine (as it should be since this currently installs PL 2.09), and push to the PR branch.

**Comment by c-bata** [comment]
Sounds good! Please proceed with the updates. Much appreciated!
